src/main/java/io/hops/util/featurestore/ops/read_ops/FeaturestoreGetMetadataForFeaturegroup.java

```diff
@@ -0,0 +1,111 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hops.util.featurestore.ops.read_ops;
+
+import io.hops.util.FeaturestoreRestClient;
+import io.hops.util.exceptions.CannotInsertIntoOnDemandFeaturegroups;
+import io.hops.util.exceptions.CannotReadPartitionsOfOnDemandFeaturegroups;
+import io.hops.util.exceptions.CannotUpdateStatsOfOnDemandFeaturegroups;
+import io.hops.util.exceptions.CannotWriteImageDataFrameException;
+import io.hops.util.exceptions.DataframeIsEmpty;
+import io.hops.util.exceptions.FeaturegroupCreationError;
+import io.hops.util.exceptions.FeaturegroupDeletionError;
+import io.hops.util.exceptions.FeaturegroupDisableOnlineError;
+import io.hops.util.exceptions.FeaturegroupDoesNotExistError;
+import io.hops.util.exceptions.FeaturegroupEnableOnlineError;
+import io.hops.util.exceptions.FeaturegroupMetadataError;
+import io.hops.util.exceptions.FeaturegroupUpdateStatsError;
+import io.hops.util.exceptions.FeaturestoreNotFound;
+import io.hops.util.exceptions.FeaturestoresNotFound;
+import io.hops.util.exceptions.HiveNotEnabled;
+import io.hops.util.exceptions.InvalidPrimaryKeyForFeaturegroup;
+import io.hops.util.exceptions.JWTNotFoundException;
+import io.hops.util.exceptions.OnlineFeaturestoreNotEnabled;
+import io.hops.util.exceptions.OnlineFeaturestorePasswordNotFound;
+import io.hops.util.exceptions.OnlineFeaturestoreUserNotFound;
+import io.hops.util.exceptions.SparkDataTypeNotRecognizedError;
+import io.hops.util.exceptions.StorageConnectorDoesNotExistError;
+import io.hops.util.exceptions.StorageConnectorNotFound;
+import io.hops.util.exceptions.StorageConnectorTypeNotSupportedForFeatureImport;
+import io.hops.util.exceptions.TrainingDatasetCreationError;
+import io.hops.util.exceptions.TrainingDatasetDoesNotExistError;
+import io.hops.util.exceptions.TrainingDatasetFormatNotSupportedError;
+import io.hops.util.featurestore.ops.FeaturestoreOp;
+
+import javax.xml.bind.JAXBException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class FeaturestoreGetMetadataForFeaturegroup extends FeaturestoreOp {
+  
+  private String[] keys;
+  
+  public FeaturestoreGetMetadataForFeaturegroup(String name) {
+    super(name);
+  }
+  
+  @Override
+  public Object read() throws FeaturestoreNotFound, JAXBException,
+      TrainingDatasetFormatNotSupportedError, TrainingDatasetDoesNotExistError,
+      IOException, FeaturestoresNotFound, JWTNotFoundException, HiveNotEnabled,
+      StorageConnectorDoesNotExistError, FeaturegroupDoesNotExistError,
+      CannotReadPartitionsOfOnDemandFeaturegroups, StorageConnectorNotFound,
+      OnlineFeaturestorePasswordNotFound, OnlineFeaturestoreUserNotFound,
+      OnlineFeaturestoreNotEnabled, FeaturegroupMetadataError {
+    if(keys == null || keys.length == 0){
```

Comment:
Your formatting is off here and in a couple of places. Could you run auto format on the new lines.

Reply:
ok